Microverstrial-3.html

```diff
@@ -0,0 +1,14 @@
+<nav id="navbar">

```

Comment:
**head** element is **missing**, it would be better if you start your html file properly.
:red_circle: **the doctype html is really important** :red_circle:

Reply:
Good 👍 job